velodrome/sql/model.go

```diff
@@ -23,8 +23,8 @@ import (
 
 // Issue is a pull-request or issue. Its format fits into the ORM
 type Issue struct {
-	Repository     string
-	ID             int
+	Repository     string `gorm:"primary_key"`
+	ID             string `gorm:"primary_key"`
```

Comment:
Oh hilarious, is that because it is the only int? Or is this because it is an int and the name is ID? Aka int would be fine if we change this to number?